src/measure_theory/constructions/polish.lean

```diff
@@ -728,6 +729,8 @@ begin
   exact this.polish_space,
 end
 
+namespace polish_space
```

Comment:
I moved the namespace up a few lines because I wanted to limit the effect of `local attribute [-instance] polish_space_of_complete_second_countable` only to the lemmas where it was needed. I need that instance in the code I added below.